_dashboards/reporting-cli.md

```diff
@@ -0,0 +1,390 @@
+
+With the Reporting CLI, you can specify options for your report in the command line. The report is sent to an email address as a PDF attachment by default. You can also request a PNG image or a CSV file with the `--formats` argument or embed the image in the email body instead of attaching it.
+
+You can download the report to the directory in which you are running the Reporting CLI, or you can email the report by specifying Amazon Simple Email Service (Amazon SES) or SMTP for the email transport option.
+
+You can connect to OpenSearch with any of the following authentication types:
+
+- **Basic** – Basic HTTP authentication. Use `-a basic`.
+- **Cognito** – Authentication through Amazon Cognito. Use `-a cognito`.
+- **SAML** – Authentication between an identity provider and a service provider. Use `-a saml`. Okta provides the SAML third-party authentication.
+- **No auth** – No authentication. Use `-a none`. Authentication defaults to No auth if the `-a` flag is not specified.
+
+To learn more about Amazon Cognito, see [What is Amazon Cognito?](https://docs.aws.amazon.com/cognito/latest/developerguide/what-is-amazon-cognito.html).
+
+<!--
+### Bypass authentication option
+
+The reporting CLI tool allows you to integrate it into your own workflow or environment so that you can bypass authentication or potential security issues. For example, if you use the Reporting CLI tool within an AWS Lambda instance, no security issues would occur as long as you run the Reporting plugin in OpenSearch Dashboards. In this case, you would use "No auth" to bypass the authentication process. To specify "No Auth" use `--auth none` in your request. Lambda users should test to make sure they can bypass access to Dashboards without credentials using No Auth.  
+
+To get a list of all options, see [Reporting CLI options](#reporting-cli-options).
+-->
+
+## Downloading and installing the Reporting CLI tool
+
+You can download and install the Reporting CLI tool from either the npm software registry or the OpenSearch.org [Download & Get Started](https://opensearch.org/downloads.html) page. Refer to the following sections for instructions.
+
+To learn more about the npm software registry, see the [npm](https://docs.npmjs.com/about-npm) documentation.
+
+### Downloading and installing the Reporting CLI from npm
+
+To download and install the Reporting CLI from npm, run the following command to initiate installation:
+
+```
+npm i @opensearch-project/opensearch-reporting-cli
+```
+
+### Downloading and installing the Reporting CLI from OpenSearch.org
+
+You can download the `opensearch-reporting-cli` tool from the OpenSearch.org [Command Line Tools](https://opensearch.org/downloads.html#command-line-tools) section.
```

Comment:
this link is not correct. it's for `opensearch-cli`.

Reply:
thanks. Could you provide correct link? or is it still in development.